web/dashboard/src/components/rolloutHistory/rolloutHistory.js

```diff
@@ -183,6 +183,8 @@ export function rolloutWidget(idx, arr, exactDate, dateLabel, rollback, env, app
     rounding = "rounded-t"
   }
 
+  console.log(builtInEnv)
```

Comment:
I don't know if this was accidentally left in it

Reply:
Good find. Thanks, removing,